.changeset/config.json

```diff
@@ -5,7 +5,6 @@
     { "repo": "apollographql/apollo-server" }
   ],
   "commit": false,
-  "fixed": [["@apollo/server", "@apollo/server-integration-testsuite"]],
```

Comment:
Not sure what this does

Reply:
(but i mean whatever happened on the other PR is fine so nbd)

Reply:
May or may not have been necessary, I flailed a bit trying to get the publish PR to only want to patch the plugin (and not the server + testsuite packages as well).